### ISSUE TITLE
 missing API

### ISSUE THREAD

**Issue by sm2017** [flutter-webrtc/flutter-webrtc#538]
I found that some APIs is missing https://github.com/flutter-webrtc/flutter-webrtc/blob/6ae3c5b84c5f2555063d5bac73da803867b1b2de/lib/src/interface/media_stream_track.dart#L117-L121

I need to get device Id of a media stream, Is there any alternative of `getSettings().deviceId` ?

**Comment by cloudwebrtc** [comment]
Yes, these are still not implemented and will be added in future versions,

**Comment by marynovskyi** [comment]
Any updates on it?
